src/kbc_modules/offline_sev_kbc/mod.rs

```diff
@@ -14,10 +14,10 @@ use std::collections::HashMap;
 use std::fs;
 use std::process::Command;
 
-const KEYS_PATH: &str = "/sys/kernel/security/coco/efi_secret/e6f5a162-d67f-4750-a67c-5d065f2a9910";
+const KEYS_PATH: &str = "/sys/kernel/security/secrets/coco/e6f5a162-d67f-4750-a67c-5d065f2a9910";
```

Comment:
How about also updating `SecretKernelModule::new()` so that the existing message (`"Failed to load secret module."`) is enhanced to mention that the user needs a 5.19+ kernel?

Reply:
But such a message might not be accurate -- we hope that distributions will backport the efi_secret module to the kernels they are shipping, which might be older than 5.19.

Reply:
I'll make sure that the README says something about 5.19, but there are a few corners where an updated message might be confusing.